studio/components/interfaces/Billing/AddOns/ComputeSizeSelection.tsx

```diff
@@ -24,8 +24,16 @@ const ComputeSizeSelection: FC<Props> = ({
   const { ref } = useParams()
   const addonUpdateDisabled = useFlag('disableProjectCreationAndUpdate')
 
+  useEffect(() => {
+    const hash = window.location.hash
+    if (hash.length > 0) {
+      window.location.hash = ''
+      window.location.hash = hash
+    }
+  })
```

Comment:
got this from here:
https://github.com/vercel/next.js/issues/11109#issuecomment-1133904021

is it a no no to use `useEffect` now?

Reply:
hmm - useEffect without dependency array looks a little iffy
cc @alaister just to sanity check

Reply:
ok added asPath as dependency. useEffect doesn't get triggered again but in this case, we don't need to trigger it again since the page is already loaded and the browser will be able to handle the jumps to the anchors?

can merge?